[2024-03-21 19:13:01,303] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T19:12:00+00:00 [queued]>
[2024-03-21 19:13:01,329] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T19:12:00+00:00 [queued]>
[2024-03-21 19:13:01,330] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 19:13:01,330] {taskinstance.py:1088} INFO - Starting attempt 1 of 6
[2024-03-21 19:13:01,330] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 19:13:01,347] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2024-03-21T19:12:00+00:00
[2024-03-21 19:13:01,360] {standard_task_runner.py:52} INFO - Started process 45683 to run task
[2024-03-21 19:13:01,365] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', '2024-03-21T19:12:00+00:00', '--job-id', '10534', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpdj0qicnm', '--error-file', '/tmp/tmpef0fy538']
[2024-03-21 19:13:01,366] {standard_task_runner.py:77} INFO - Job 10534: Subtask retrieve_raw_data
[2024-03-21 19:13:01,435] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.retrieve_raw_data 2024-03-21T19:12:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 19:13:01,531] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=retrieve_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T19:12:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T19:12:00+00:00
[2024-03-21 19:13:02,095] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 19:13:02,108] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20240321T191200, start_date=20240321T191301, end_date=20240321T191302
[2024-03-21 19:13:02,148] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-03-21 19:13:02,181] {local_task_job.py:151} INFO - Task exited with return code 0
